src/app/pages/company/company.component.ts

```diff
@@ -1,10 +1,11 @@
-import {Component, OnInit} from '@angular/core';
-import {FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
-import {Router} from '@angular/router';
-import {CompanyService} from '../../services/company/company.service';
-import {Company} from '../../models';
-import {first} from 'rxjs/operators';
-import {ToastrService} from 'ngx-toastr';
+import { Meta, Title } from '@angular/platform-browser';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+import { CompanyService } from '../../services/company/company.service';
+import { Company } from '../../models';
```

Comment:
The `Company` model import would be a good candidate for absolute import.

Reply:
Thanks, @Victoradukwu, I have worked on this.